js/views/modals/About/About.js

```diff
@@ -0,0 +1,100 @@
+import $ from 'jquery';
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseModal from '../BaseModal';
+import Story from './Story';
+import Contributors from './Contributors';
+import Donations from './Donations';
+import License from './License';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    const opts = {
+      removeOnClose: true,
+      removeOnRoute: false,
+      ...options,
+    };
+
+    super(opts);
+    this.options = opts;
+
+    this.tabViewCache = {};
+    this.tabViews = {
+      Story,
+      Contributors,
+      Donations,
+      License,
+    };
+
+    this.listenTo(app.router, 'will-route', () => {
+      this.close(true);
+      this.remove();
+    });
+  }
+
+  className() {
+    return `${super.className()} about tabbedModal modalTop modalScrollPage`;
+  }
+
+  events() {
+    return {
+      'click .js-tab': 'tabClick',
+      ...super.events(),
+    };
+  }
+
+  get closeClickTargets() {
+    return [
+      ...this.$closeClickTargets.get(),
+      ...super.closeClickTargets,
+    ];
+  }
+
+  tabClick(e) {
+    const targ = $(e.target).closest('.js-tab');
+
+    this.selectTab(targ);
+  }
+
+  selectTab(targ) {
+    const tabViewName = targ.data('tab');
+    let tabView = this.tabViewCache[tabViewName];
+
+    if (!this.currentTabView || this.currentTabView !== tabView) {
+      this.$('.js-tab').removeClass('clrT active');
+      targ.addClass('clrT active');
+      if (this.currentTabView) this.currentTabView.$el.detach();
+
+      if (!tabView) {
+        tabView = this.createChild(this.tabViews[tabViewName]);
+        this.tabViewCache[tabViewName] = tabView;
+        this.listenTo(tabView, 'saving', (...args) => { this.onTabSaving(tabView, ...args); });
```

Comment:
This line can be removed. It's specific to the settings modal where tabs were communicating their 'saving' state to the parent. It doesn't apply here.